src/data_capture/video_capture/launch/capture.launch

```diff
@@ -12,6 +13,7 @@
         args="
             --image-topic $(arg image_topic) 
             --is-record-topic $(arg is_record_topic)
+            --is-memory-usage-exceeded-topic $(arg is_memory_usage_exceeded_topic)
```

Comment:
It would be nice to refactor `capture.py` to use ROS parameters instead of args.